catalog/plugin/catalog-plugin-videothumbnail/pom.xml

```diff
@@ -127,9 +127,15 @@
                                 <rule>
                                     <element>BUNDLE</element>
                                     <limits>
+                                        <!--
+                                        The coverages here have been lowered to 0 because coverage
+                                        is OS-specific. OS X coverage ratios have been left
+                                        commented out for reference.
```

Comment:
@stustison, do you remember why? https://github.com/codice/ddf/commit/eab606161861ba3534f05e5ffdc167d53c17668a